Dockerfile

```diff
@@ -0,0 +1,32 @@
+# To build one auto-instrumentation image for dotnet, please:
+#  - Download your dotnet auto-instrumentation artefacts to `/autoinstrumentation` directory. This is required as when instrumenting the pod,
+#    one init container will be created to copy the files to your app's container.
+#  - Grant the necessary access to the files in the `/autoinstrumentation` directory.
```

Comment:
Are the necessary rights the last line of the Dockerfile? Perhaps we should list here what are the necessary rights and to which files.

Reply:
If you really think that is worth to do, we should open PR in the upstream repository first.

Reply:
This seems worth to follow-up just to be explicit about it, but, low priority.